ui/app/modifiers/code-mirror.js

```diff
@@ -25,8 +25,15 @@ export default class CodeMirrorModifier extends Modifier {
     }
   }
 
-  didInstall() {
-    this._setup();
+  element = null;
+  args = {};
+
+  modify(element, positional, named) {
+    if (!this.element) {
+      this.element = element;
+      this.args = { positional, named };
+      this._setup();
+    }
```

Comment:
An update to our design systems library also brought about an update to our code editor library. This `modify` syntax is an update to the older `didInstall` syntax we'd been using before.